gxcloudstorage-common/src/main/java/com/genexus/db/driver/ExternalProviderBase.java

```diff
@@ -14,9 +14,7 @@ public abstract class ExternalProviderBase {
 	static final String DEFAULT_ACL = "DEFAULT_ACL";
 	static final String DEFAULT_EXPIRATION = "DEFAULT_EXPIRATION";
 	static final String FOLDER = "FOLDER_NAME";
-
```

Comment:
Please Revert the Property Name for compatibility reasons

Reply:
Done